enable/vtk_backend/vtk_window.py

```diff
@@ -281,6 +283,7 @@ def _vtk_mouse_wheel(self, vtk_obj, eventname):
             self._pass_event_to_vtk(vtk_obj, eventname)
 
     def _create_key_event(self, vtk_event, event_type):
+        # FIXME: THIS IS A BUG
```

Comment:
Do you think we can just replace the variables with `vtk_event` and `event_type`? That looks right from here, but there is no test to check if that works.